frontend/src/App.tsx

```diff
@@ -678,20 +747,43 @@ function App() {
             </Tooltip>
 
             {accounts.length === 0 && (
-              <Typography style={{ color: 'gray', margin: '20px' }}>
-                No accounts found
-              </Typography>
+              <>
+                <Typography style={{ color: 'black', margin: '20px' }}>
+                  No accounts found
+                </Typography>
+                <Typography style={{ color: 'gray', margin: '20px' }}>
+                  To add account to this list, click on "Remember me" checkbox
+                  before. Accounts do not know about your connection preferences.
+                  login.
```

Comment:
What is this last "login"?

Reply:
the one succesfully logged in, not stored in localstorage,